djangoprojects/provision.py

```diff
@@ -48,43 +52,44 @@ def create(droplet_name):
              "region": "nyc3",
              "size": "s-1vcpu-1gb",
              "image": "ubuntu-16-04-x64",
-             "ssh_keys": [23341984],
+             "ssh_keys": [fingerprint.hash_md5().replace("MD5:", "")],
              "backups": False,
              "ipv6": True,
              "user_data": None,
              "private_networking": None,
              "volumes": None,
              "tags": [
-                   "Name:"+droplet_name
+                   "name:"+droplet_name
                  ]
             }
-    api_url = '{0}droplets'.format(api_url_base)
-    response = requests.post(api_url, headers=headers, json=data)
-    time.sleep(25)  # delays for 25 seconds
-    if response.status_code == 202:
-        print("SUCCESSFULLY CREATED")
-        jsondata = json.loads(response.text)
-        response_id = (jsondata.get("droplet")).get("id")
-        response_status = (jsondata.get("droplet")).get("status")
-        data = {'id': response_id, 'status': response_status}
-    else:
-        print("FAILED TO CREATE")
-    return data
+    api_url = '{0}droplets'.format(API_URL_BASE)
+    response = requests.post(api_url, headers=HEADERS, json=data)
+    jsondata = response.json()
+    droplet = jsondata['droplet']
+    while True:
```

Comment:
loop is to wait till droplet is active, since we are later logging in and running commands to it